cmd/node/factory/structs.go

```diff
@@ -1829,6 +1830,7 @@ func newShardBlockProcessor(
 	rounder consensus.Rounder,
 	statisticsProcessor process.ValidatorStatisticsProcessor,
 	bootStorer process.BootStorer,
+	gasSchedule map[string]map[string]uint64,
```

Comment:
Like my previous comment. This should be a type. It will be very hard for anyone else to figure out how to use this

Reply:
i will make a task for this in current sprint.